ACCEPTANCE_TESTS.rst

```diff
@@ -0,0 +1,163 @@
+S3 Acceptance Tests
+===================
+
+List of all of the various scenarios that need to be handled in implementing
+a S3 transfer manager.
+
+Upload Tests
+------------
+
+General
+~~~~~~~
+* [x] Upload single nonmultipart file
+* [x] Upload single multipart file
+* [x] Upload multiple nonmultipart files
+* [x] Upload multiple multipart files
+* [x] Failed/cancelled multipart upload is aborted and leaves no orphaned parts
+* [ ] Ctrl-C of any upload does not hang
```

Comment:
In general I agree. It would be nice to have a even more aggressive time than the CLI has in its tests.  Right now, I think the CLI allows up to 30 seconds to exit.